nolooking/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       PROXY_AUTH_WHITELIST: "/pj"
   
   server:
-    image: chaincase/nolooking:v0.2.2-alpha@sha256:e4a95443a05b44659f55a20c592142d59a4d8f54830a71f4c3d22a3a6bf88801
+    image: chaincase/nolooking:v0.3.0-alpha@sha256:ba879c089e6609cb85869081c18bc36086984276c5cd73bce2a24ff5f53cc1f0
     restart: on-failure
```

Comment:
I had an issue on 1 of my Umbrels with LND causing nolooking to crash. It looks like when you exit, it's exiting with a success status code (`0`) which means the container's restart policy (`no-failure`) will not be applied, and therefore won't automatically restart the container when something goes wrong.

You can see from the error I got, that LND was still booting, and so a auto-restarting container would be quite effective. You could also however, solve this in your app, but having a 'retry loop' for certain errors.
```
Running nolooking
Error: Lnd(VersionRequestFailed(Status { code: Unknown, message: "the RPC server is in the process of starting up, but not yet ready to accept calls", metadata: MetadataMap { headers: {"content-type": "application/grpc"} }, source: None }))
nolooking exited!
```

Reply:
Aw rats. Must be because the Dockerfile CMD starts nolooking using `run_nolooking` shell script instead of calling the binary directly, which exits with a non-zero code in the scenario you described. Building a new docker image to fix!